### PR TITLE
Cut the first few paragraphs down, by quite a lot

### PR DESCRIPTION
**Pull request by pavpanchekha**
These are some pretty extreme cuts for the first few paragraphs. Totally willing to pare them back!

### CODE REVIEW

File: book/browsersandweb.md [hunk 1]
```diff
@@ -5,71 +5,52 @@ next: http
...
-
-Figuring out how to meet all these lofty goals with real code is certainly
-difficult, but _also_ groundbreaking, endlessly fascinating even in the details,
-and often  just plain fun. That's what this book is about.
+    [web pages](https://en.wikipedia.org/wiki/Web_page) on the Internet.
+
+Nowadays, desktop applications are often built and delivered as _web
+apps_: websites[^website] used in similar ways to installed
+applications. And on mobile devices even native apps often use _web
+views_ that embed a browser render parts of the application UI.
```
Comment (chrishtr) [inline book/browsersandweb.md:20]:
```suggestion
views_ that embed a browser to render parts of the application UI.
```

File: book/browsersandweb.md [hunk 2]
```diff
@@ -5,71 +5,52 @@ next: http
...
+Perhaps in the future mobile devices will, like desktop computers,
+mostly a container for web apps.
```
Comment (chrishtr) [inline book/browsersandweb.md:22]:
```suggestion
mostly be a container for web apps.
```

File: book/browsersandweb.md [hunk 3]
```diff
@@ -5,71 +5,52 @@ next: http
...
+    If not, [see here](https://en.wikipedia.org/wiki/Website).
+
+[^hybrid]: The fraction of these _hybrid apps_ that are web content is
+    also likely increasing over time.
+
+Yet this critical piece of software is built on a few simple, yet
+revolutionary, concepts that present a vision of the future of
+software and information: open, decentralized and safe computing;
+declarative UIs; hyperlinks; and user agents[^useragent]. This vision
+is neither simple nor obvious, even if it seems that way in hindsight:
```
Comment (chrishtr) [inline book/browsersandweb.md:34]: Here you changed the argument to emphasize the other technologies that made the web possible, but skipped the part about prior research into information systems. Was that on purpose?

**Review by chrishtr — COMMENTED** [review]
Thank you for the suggested edits! I like a lot of what you did here. Comments inline as well.

File: book/browsersandweb.md (hunk 3 above)
Comment (pavpanchekha) [reply book/browsersandweb.md:34]: Yeah, I think I did not do this well. The real thought here was to keep the focus on browsers, whereas this paragraph changes focus to the web and concepts that aren't directly related to it. I tried swapping parts between this and the next paragraph, and that didn't really work, so I kind of ended up with this.

The idea of focusing on browsers still seems good but I think you can do a better job than I did.

Comment (chrishtr) [reply book/browsersandweb.md:34]: Ok. Revert this part or I'll try to patch in parts of your PR?

Comment (chrishtr) [reply book/browsersandweb.md:34]: You make a good point about the change of focus. Here is another take on resolving this issue. Better?

https://github.com/pavpanchekha/emberfox/pull/53

**Comment by pavpanchekha** [comment]
I merged in your changes in #53 and am going to do another pass.

**Comment by pavpanchekha** [comment]
I did another attempt at those last few paragraphs. Not that I wasn't trying to refocus it, I think it hews much closer to the original, inverting a few sentences to try to shorten it. It still might be too aggressive, so please feel free to either extract what you like into a new PR or push commits onto this one to pare some edits back.

File: book/browsersandweb.md [hunk 4]
```diff
@@ -5,72 +5,57 @@ next: http
...
+information. Among them are open, decentralized and safe computing; a
+declarative document model for describing UIs; hyperlinks; and the
+User Agent concept[^useragent]. Since the browser makes the web real,
+all these concepts form the core structure of the browser code itself.
+
+[^useragent]: The User Agent is a way to view the computer, or
+    software within the computer, as a trusted assistant and advocate.
+
+This vision of the web is neither simple nor obvious; it is the result
+ofexperimentation and research reaching back to nearly the beginning
```
Comment (chrishtr) [inline book/browsersandweb.md:42]:
```suggestion
of experimentation and research reaching back to nearly the beginning
```

File: book/browsersandweb.md [hunk 5]
```diff
@@ -5,72 +5,57 @@ next: http
...
+of computing. And the web _also_ needed rich computer displays,
+powerful UI-building libraries, fast consumer networks, and sufficient
+CPU power and information storage capacity. As so often happens, the
+web has many predecessors but only took its modern form in the late
+1990s.
```
Comment (chrishtr) [inline book/browsersandweb.md:47]:
```suggestion
1980s, once all those technologies were available.
```

File: book/browsersandweb.md [hunk 6]
```diff
@@ -5,72 +5,57 @@ next: http
...
-apps_---websites[^website] that are intended to be used in similar ways to
-installed applications, not just documents. Many people now spend their entire
-day in a browser, not using a single other application.
-
-[^webbrowser]: Henceforth, simply “browser”.
+The web browser, and the web[^theweb] more broadly, is a marvel. Every
+year it expands its reach to more and more of what we do with
+computers. It now goes far beyond its original use for document-based
+information sharing; many people now spend their entire day in a
+browser, not using a single other application.
```
Comment (chrishtr) [inline book/browsersandweb.md:12]:
```suggestion
browser, not using a single other application!
```

File: book/browsersandweb.md [hunk 7]
```diff
@@ -5,72 +5,57 @@ next: http
...
+Nowadays, desktop applications are often built and delivered as _web
+apps_: websites[^website] used in similar ways to installed
+applications. And on mobile devices even native apps often use _web
+views_ that embed a browser to render parts of the application UI.
+Perhaps in the future mobile devices will, like desktop computers,
+mostly be a container for web apps.
 
-[^website]: You probably already know what a website is. If not, [see
-here](https://en.wikipedia.org/wiki/Website).
+[^website]: You probably already know what a website is. This is one.
```
Comment (chrishtr) [inline book/browsersandweb.md:24]:
```suggestion
[^website]: You probably already know what a website is. You're  probably reading this book on one right now!
```

File: book/browsersandweb.md [hunk 8]
```diff
@@ -5,72 +5,57 @@ next: http
...
-web would be no fun if websites were not fast to load and smooth to use, so the
-browser must be hyper-efficient as well.
-
-Lofty goals! How does the browser deliver on them? It's a fascinating and fun
-journey. That's what this book is about.
+itself is built on a few simple, yet revolutionary, concepts; concepts
+that that together present a vision of the future of software and
+information. Among them are open, decentralized and safe computing; a
+declarative document model for describing UIs; hyperlinks; and the
+User Agent concept[^useragent]. Since the browser makes the web real,
```
Comment (chrishtr) [inline book/browsersandweb.md:35]:
```suggestion
User Agent[^useragent]. Since the browser makes the web real,
```

File: book/browsersandweb.md [hunk 9]
```diff
@@ -5,72 +5,57 @@ next: http
...
+all these concepts form the core structure of the browser code itself.
+
+[^useragent]: The User Agent is a way to view the computer, or
+    software within the computer, as a trusted assistant and advocate.
+
+This vision of the web is neither simple nor obvious; it is the result
+of experimentation and research reaching back to nearly the beginning
+of computing. And the web _also_ needed rich computer displays,
```
Comment (chrishtr) [inline book/browsersandweb.md:43]:
```suggestion
of computing. And of course the web _also_ needs rich computer displays,
```

**Comment by chrishtr** [comment]
LGTM! I just fixed some nits and added one more phrase. Committing.

Looking better and better!

**Pull request merged**